File: app/src/main/java/com/tebet/mojual/view/qualitycheck/QualityFragmentProvider.java
package com.tebet.mojual.view.qualitycheck;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
public abstract class QualityFragmentProvider {

    @ContributesAndroidInjector(modules = QualityFragmentModule.class)
    abstract QualityFragment provideQualityFragmentFactory();
}

File: app/src/main/java/com/tebet/mojual/view/qualitycheck/QualityFragmentProvider.kt
package com.tebet.mojual.view.qualitycheck

import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module
abstract class QualityFragmentProvider {

    @ContributesAndroidInjector(modules = [QualityFragmentModule::class])
    internal abstract fun provideQualityFragmentFactory(): QualityFragment
}
